app/styles/components/course-search-result.scss

```diff
@@ -0,0 +1,31 @@
+.course-search-result {
+  margin-bottom: 1rem;
+  max-width: 40rem;
+
+  .matched-in {
```

Comment:
is `matched-in` different from `global-search-tags`?

Reply:
This needs to be removed.